src/udev/udev-builtin-net_driver.c

```diff
@@ -0,0 +1,36 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "alloc-util.h"
+#include "device-util.h"
+#include "errno-util.h"
+#include "ethtool-util.h"
+#include "fd-util.h"
+#include "log.h"
+#include "string-util.h"
+#include "udev-builtin.h"
+
+static int builtin_net_driver_set_driver(UdevEvent *event, int argc, char **argv, bool test) {
+        sd_device *dev = ASSERT_PTR(ASSERT_PTR(event)->dev);
+        _cleanup_close_ int ethtool_fd = -EBADF;
```

Comment:
Maybe, it is better to 'share' ethtool_fd with net_setup_link builtin. But not a strong opinion. Feel free to keep it.

Reply:
Yeah, makes sense. But I think that should go to a separate patch (maybe even PR?).

Reply:
Yeah, let's do that in another PR.